tests/performance_tests/performance_test_basic.py

```diff
@@ -14,33 +15,68 @@
 from Node import Node
 from Node import ReturnType
 from TestHelper import TestHelper
+from dataclasses import dataclass
+from typing import List
 
 Print = Utils.Print
 errorExit = Utils.errorExit
 cmdError = Utils.cmdError
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+@dataclass
+class blockData():
+    blockId: str = ""
+    blockNum: int = 0
+    transactions: int = 0
+    net: int = 0
+    cpu: int = 0
+    elapsed: int = 0
+    time: int = 0
+    latency: int = 0
+
+class chainData():
+    def __init__(self):
+        self.blockLog = []
+        self.startBlock = 0
+        self.ceaseBlock = 0
+        self.totalTransactions = 0
+        self.totalNet = 0
+        self.totalCpu = 0
+        self.totalElapsed = 0
+        self.totalTime = 0
+        self.totalLatency = 0
+    def updateTotal(self, block):
+        print("Block is ", block)
+        self.totalTransactions += int(block[2])
```

Comment:
Remove redundant conversions to int.  Since all of these members have been initialized to integer values, anyone inadvertently passing in unconverted strings will promptly get a TypeError exception, not an unexpectedly long string.